src/__tests__/pet.test.js

```diff
@@ -0,0 +1,168 @@
+const { Pet } = require ('../pet.js');
+
+const ERROR_MESSAGE = 'Your pet is no longer alive :(';
+
+let fido;
+beforeEach(() => {
+    fido = new Pet('Fido');
+});
+
+describe ('Pet', () => {
+
+  test('Returns an instance of Pet', () => {
+    expect(fido).toBeInstanceOf(Pet);
+  });
+
+  test('Returns the pets name', () => {
+    expect(fido.name).toBe('Fido');
+  });
+});
+
+describe ('growUp', () => {
+  
+  test ('has an initial age of 0', () => {
+    expect (fido.age).toEqual (0);
+   });
+
+   test ('increments the age by 1', () => {
+    fido.growUp();
+    expect (fido.age).toEqual (1);
```

Comment:
You don't need the space between the expect and the opening bracket here. It is standard practice not to have a space but if it makes it easier for you to read whilst you are learning, it won't break your code.